src/client/cmd/info.cpp

```diff
@@ -94,6 +101,38 @@ std::ostream& operator<<(std::ostream& out, const multipass::InfoReply::Info& in
     }
     return out;
 }
+
+QString toJson(mp::InfoReply& reply)
+{
+    QJsonObject info_json;
+
+    for (const auto& info : reply.info())
+    {
+        QJsonObject instance_info;
+        instance_info.insert("State", QString::fromStdString(status_string(info.status())));
+        instance_info.insert("IPv4", QString::fromStdString(info.ipv4()));
```

Comment:
IPv6 missing here?

Reply:
We don't support IPv6 yet.  It had also been removed from the table output as well.